packages/ui/form/PhoneInput.tsx

```diff
@@ -1,29 +1,21 @@
-import BasePhoneInput, { Props } from "react-phone-number-input/react-hook-form";
+import BasePhoneInput, { Props } from "react-phone-number-input";
```

Comment:
Can we lazy load this component?

Reply:
Yup should be doable. Raised a separate issue.